config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -116,11 +116,10 @@ spec:
       ironicNeutronAgent:
         replicas: 1
       secret: osp-secret
-  telemetry:
+  ceilometer:
     template:
-      ceilometerCentral:
-        passwordSelector:
-          service: CeilometerPassword
-        secret: osp-secret
-        serviceUser: ceilometer
-      rabbitMqClusterName: rabbitmq
+      serviceAccount: telemetry-operator-telemetry
```

Comment:
This should be created by the Telemetry operator itself, so I think we should remove it